tests/core/test_variable.py

```diff
@@ -0,0 +1,588 @@
+"""Unit tests for core.variable module."""
+import unittest
+
+import numpy as np
+
+import relentless
+
```

Comment:
Can you add a short test for the IndependentVariable class? Just that it can be initialized and have its value set and changed.